test/tests/functional/pbs_test_tpp.py

```diff
@@ -384,18 +382,22 @@ def common_setup(self):
         """
         mom_list = copy.copy(self.moms.values())
         comm_list = copy.copy(self.comms.values())
-        i = 0
-        for mom in self.moms.values():
+        _server = self.server.shortname
+        skip_test = [False if _server == mom.shortname else
+                     True for mom in self.moms.values()] and \
+                    [False if _server == comm.shortname else True for
+                     comm in self.comms.values()]
+        if skip_test[0]:
```

Comment:
Why not just put this condition in the if statement itself?  Why make the long and complex list?